### DISCUSSION TITLE
How to compare the inference speed before and after conversion

### DISCUSSION THREAD

**Discussion by ChenZhenGui** [open-mmlab/mmdeploy#1525] [Q&A]
I would like to know how to compare the inference speed of the pytorch model and the model after converting to tensorrt

**Comment by lvhan028** [comment]
Please try `tools/profiler.py`

**Comment by ChenZhenGui** [comment]
so what's the difference between test.py and profiler.py?

**Reply by lvhan028** [reply]
`profiler.py` is especially for speed test.
The main purpose of `test.py` is to check the metrics consistency between pytorch models and backend models
